01-worker.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/bash
+set -o nounset -o errexit
+
+echo --node-ip=${NODE_PRIVATE_IP} >> /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
```

Comment:
here you are writing a flag to the service file, when it actually needs to be set in the enviornment per [the documentation](https://kubernetes.io/docs/setup/independent/troubleshooting-kubeadm/). I believe the correct code to be: `echo Environment=KUBELET_EXTRA_ARGS=--node-ip=${NODE_PRIVATE_IP} >> /etc/systemd/system/kubelet.service.d/10-kubeadm.conf`